### ISSUE TITLE
java.lang.NoClassDefFoundError: com/google/protobuf/ExtensionRegistryLite

### ISSUE THREAD

**Issue by xuwei-k** [scalapb/ScalaPB#349]
build.sbt

```scala
PB.targets in Compile := Seq(scalapb.gen() -> (sourceManaged in Compile).value)

scalaVersion := "2.12.3"
```

project/build.properties 

```
sbt.version=0.13.16
```

project/plugins.sbt

```scala
addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.12")

libraryDependencies += "com.trueaccord.scalapb" %% "compilerplugin" % "0.6.5"
```

`sbt compile`

```
java.lang.NoClassDefFoundError: com/google/protobuf/ExtensionRegistryLite
	at scalapb.package$.gen(package.scala:11)
	at $e92b96a5603b73df11fa$$anonfun$$sbtdef$1.apply(/Users/kenji/scalapb-test/build.sbt:1)
	at $e92b96a5603b73df11fa$$anonfun$$sbtdef$1.apply(/Users/kenji/scalapb-test/build.sbt:1)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
	at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
	at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
	at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
	at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: com.google.protobuf.ExtensionRegistryLite
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at scalapb.package$.gen(package.scala:11)
```

**Comment by nktpro** [comment]
Same here, only since latest 0.6.5 release.

**Comment by thesamet** [comment]
Fixed in 0.6.6.

**Comment by nktpro** [comment]
Thanks @thesamet you rock as always!
